CMakeLists.txt

```diff
@@ -103,7 +127,12 @@ CHECK_INCLUDE_FILES("sys/types.h" HAVE_SYS_TYPES_H)
 CHECK_INCLUDE_FILES("sys/vfs.h" HAVE_SYS_VFS_H)
 CHECK_INCLUDE_FILES("sys/prctl.h" HAVE_SYS_PRCTL_H)
 CHECK_INCLUDE_FILES("execinfo.h" HAVE_EXECINFO_H)
-CHECK_INCLUDE_FILES("sched.h" HAVE_SCHED)
+if(LINUX)
+  CHECK_INCLUDE_FILES("sched.h" HAVE_SCHED)
+endif(LINUX)
+if(FREEBSD)
+  CHECK_INCLUDE_FILES("sched.h" HAVE_SCHED_FREEBSD)
```

Comment:
@tchaikov 
Not yet, but sched.h is detected on FreeBSD, but is not compatible.
And that needs work. So either it is just excluded, or we already make a note, and work on the scheduler when the more pressing things are fixed.